handsontable/src/core.js

```diff
@@ -3594,21 +3594,44 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
   };
 
   /**
-   * Returns an array of column headers (in string format, if they are enabled). If param `column` is given, it
-   * returns the header at the given column.
+   * Gets the values of column headers (if column headers are [enabled](@/api/options.md#colheaders)).
    *
-   * Since the 12.3.0 the method accepts the 2nd `headerLevel` argument.
+   * To get an array with the values of all column headers, call `getColHeader()` with no arguments.
+   *
+   * To get the value of a specific column header , use the `column` parameter.
+   *
+   * To get the values of column headers at [a specific level](@/guides/columns/column-groups.md),
+   * use the `headerLevel` parameter.
+   *
+   * Read more:
+   * - [Guides: Column groups](@/guides/columns/column-groups.md)
+   * - [Options: `colHeaders`](@/api/options.md#colheaders)
+   * - [Guides: Copy with headers](@/guides/cell-features/clipboard.md#copy-with-headers)
+   *
+   * ```js
+   * // get the contents of all column headers
+   * hot.getColHeader();
+   *
+   * // get the contents of a specific column header
```

Comment:
```suggestion
   * // get the contents of a specific most-bottom column header
```
Do you think it's worth adding information that you are getting (by default) the most-bottom header value?

Reply:
Definitely, done: [25e0e9e](https://github.com/handsontable/handsontable/pull/10118/commits/25e0e9e161f694c2a9cacd94e905ec8317614ad2)